app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class RegistrationsController < Devise::RegistrationsController
+  DEFAULT_REGISTRATION_PARAMS = { active: false, self_registered: true }.freeze
+  PERMITTED_PARAMS = %i[email full_name cellphone cpf address].freeze
+
+  before_action :configure_permitted_parameters
+  after_action :send_user_self_registration_notification_email, only: %i[create] # rubocop:disable Rails/LexicallyScopedActionFilter
```

Comment:
@m-pereira I disabled this Rubocop rule for this case, is this ok?

The reason is that, at first Rubocop pointed out that [create is not defined in the class](https://github.com/comarev/comarev/runs/5787518448?check_suite_focus=true#step:4:12), but if I explicitly add the method, like so:
```rb
def create
  super
end
```

Rubocop complains about an useless method:
```rb
app/controllers/registrations_controller.rb:10:3: W: [Correctable] Lint/UselessMethodDefinition: Useless method definition detected.
```

Reply:
I think that's ok commenting as you did, it's easy to find this on the rubocop docs